### ISSUE TITLE
Tabs/Tab - missing padding between Tab Control & Tab Content

### ISSUE THREAD

**Issue by cstrat** [mantinedev/mantine#126]
Question on `Tabs`, there previously was padding between the tab bar and the tab content.
Do we now need to use the Styles API to inject some padding, or is the below a bug.

No padding at all - which is strange because even in the docs there is padding.
See the highlighted text butts up against the outline tab bar.
This is happening across all the tabs in my app (using mix of default and outline variants).

My App (seems broken):
<img width="186" alt="image" src="https://user-images.githubusercontent.com/4209076/124525751-bc676580-de43-11eb-9563-c251daed47f0.png">

Mantin Docs (works correctly!):
<img width="158" alt="image" src="https://user-images.githubusercontent.com/4209076/124525873-3992da80-de44-11eb-812b-6ae5a67e34ab.png">

The immediate child inside the `<Tab>` is a `<form>` - but not sure why that would make a difference.
Other tabs which are affected have `<Text>` or vanilla `<div>` as the immediate children.

Code for my screenshot:

```js
<div style={{ position: 'relative' }}>
  <LoadingOverlay visible={loading} />
  <Tabs variant="outline">
    <Tab
      disabled={disableTabs}
      icon={<FontAwesomeIcon icon={faAddressCard} />}
      label="Profile"
    >
      <form>...</form>
    </Tab>
  </Tabs>
</div>
````

**Comment by cstrat** [comment]
Tested this across both Chrome and Safari.
Tested in incognito window and ensured no cache.

**Comment by cstrat** [comment]
Example:
https://codesandbox.io/s/mantine-tabs-padding-3zpnc?file=/src/App.js

Docs:
<img width="244" alt="image" src="https://user-images.githubusercontent.com/4209076/124538552-30fbcd80-de5f-11eb-87bc-2bffa8f769c9.png">


Example:
<img width="213" alt="image" src="https://user-images.githubusercontent.com/4209076/124538580-3ce78f80-de5f-11eb-8260-b6228f54e8c6.png">



If the idea is that each `<Tab>` adds its own padding, that should be explained?
Although it makes more sense that the `<Tabs>` has a padding offset to the content below.
I tried but can't even manually add this with `style={{ padding: 16 }}`.

**Comment by rtivital** [comment]
By default Tabs do not have padding-top to allow [something like this](https://mantine.dev/others/prism/#languages), my idea was that you provide padding on children:

```tsx
<Tabs>
  <Tab><div style={{ paddingTop: 20 }}>My children</div></Tab>
</Tabs>
```

But I see, how it can be confusing, I'll add `tabPadding` prop to Tabs component in 2.0.2

**Comment by cstrat** [comment]
I only noticed because there previously was padding in 1.3.
Happy if that's the way forward but it would be good if we didn't need to add padding to each `Tab` but rather once in the `Tabs` element. So `tabPadding` in `Tabs` would be ideal.

1.3
![image](https://user-images.githubusercontent.com/4209076/124549435-7fff2e00-de72-11eb-9437-c1fde3651181.png)

2.0
![image](https://user-images.githubusercontent.com/4209076/124549473-8ab9c300-de72-11eb-9d60-131df4e96583.png)

https://codesandbox.io/s/mantine-tab-padding-zqo3e?file=/src/App.js -- flick between the versions.

**Comment by rtivital** [comment]
Yes, that's changed in 2.0.0, I forgot to put it in changelog, anyway I'll add `tabPadding` prop and it will be the same as in 1.3 by default

**Comment by cstrat** [comment]
Thanks again for the great release!

**Comment by rtivital** [comment]
Resolved with #135
